src/plugins/intel_gna/src/ops/util/util.hpp

```diff
@@ -84,16 +99,30 @@ static bool is_aligned_split(const std::shared_ptr<ngraph::Node> input_op, size_
     if (std::dynamic_pointer_cast<ngraph::opset8::Split>(input_op) || std::dynamic_pointer_cast<ngraph::opset8::VariadicSplit>(input_op)) {
         for (size_t index = 0; index < input_op_out_index; index++) {
             size_t outputSize = ngraph::shape_size(input_op->get_output_shape(index));
-            offset += outputSize * GNAPluginNS::GNALimitations::bytesPerSplitElement;
+            offset += outputSize * bytesPerSplitElement;
         }
     }
     return (offset == ALIGN64(offset));
 }
 
+static bool is_split_supported(const std::shared_ptr<ov::Node>& node, bool is_exception_allowed) {
+    if (!node) {
+        if (is_exception_allowed) {
+            THROW_GNA_EXCEPTION << "Split node is empty!\n";
+        }
+        return false;
+    }
+    bool is_aligned = true;
+    for (size_t i = 0; i < node->get_output_size(); i++) {
+        is_aligned &= is_aligned_split(node, i);
+    }
+    return is_aligned;
+}
+
 static bool is_crop_affined(std::shared_ptr<ngraph::Node> node) {
     auto crop = std::dynamic_pointer_cast<ngraph::op::CropIE>(node);
     if (crop != nullptr && !crop->offset.empty()) {
-        return GNAPluginNS::GNALimitations::isCropAffinedOffset(crop->offset.back());
```

Comment:
It is header
I would revert change or add using namespace at the beginning of the method.

http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-using-directive